### ISSUE TITLE
exclude sub-package or objects 

### ISSUE THREAD

**Issue by simoper** [larshp/abapGit#1115]
Sometimes a sub-package or several objects should be excluded from the repository so that this package and objects will be ignored. F. e. moving the unwanted package or objects somewhere else in the package hierarchy is not possible.
[https://abapgit.slack.com/archives/C4K5GM9D0/p1513849170000304](https://abapgit.slack.com/archives/C4K5GM9D0/p1513849170000304)

**Comment by larshp** [comment]
Guess this setting should be something stored locally, and not part of .abapgit.xml

**Comment by larshp** [comment]
Following user exit can be used to modify the objects found in TADIR, gues this solves the issue?

![image](https://user-images.githubusercontent.com/5888506/44190546-99be9280-a127-11e8-9a07-c902c537e210.png)

**Comment by CASAG-MariusRaht** [comment]
@larshp Hi, why is this issue closed? Is there a working solution for it I didn't find? The named user exit is apparently very suitable and I tested a self-developed solution for excluding packages. For that I extended the repository settings page so one can maintain packages to be excluded and filter the internal tadir-table in the user exit you mentioned.

My tests with one package were successful and I have to extend my solution only for the excluding of multiple packages. There's just one problem with the Online-Repository-Popup I experienced so far: I couldn't find a suitable table and field for the new field "Excl. Pkgs." in zcl_abapgit_popups~repo_popup and chose SVAL-VALUE (max. 132 characters); I'd like to have 255 chars - is there a better table-field-combination you know of?
